README.md

```diff
@@ -3,3 +3,15 @@
 A tool for tracking blogs in orbit around Seneca's open source involvement
 
 An initial discussion of the project is available in the [Overview](docs/overview.md) document.
+
+#Get the Code
```

Comment:
Space after `#`

Reply:
Let's also mention that this is for the legacy code vs. what we're building.

Reply:
Adding on to @humphd's comment, we should include that the wget command is for Linux/Mac and doesn't work for Windows. I ran this on Fedora but for some reason the --no-parent option didn't do what it was suppose to do so I ran:
# wget -r --no-parent https://people.gnome.org/~jdub/bzr/planet/devel/trunk/
# mv people.gnome.org/~jdub/bzr/planet/devel/trunk cdotplanet
# cd cdotplanet

Reply:
The requested changes have been made. 
Please LMK if there are any issues @humphd